lib/container/client.go

```diff
@@ -0,0 +1,59 @@
+package container
+
+import (
+	"fmt"
+	"os/exec"
+
+	credClient "github.com/docker/docker-credential-helpers/client"
+)
+
+var (
+	client   Client
+	binaries = []RuntimeBinary{
+		RuntimeBinaryNerdctl,
+		RuntimeBinaryPodman,
+		RuntimeBinaryDocker,
+	}
+)
+
+var rt Runtime
+
+type Client struct {
+	runtimePath string
+}
+
+func InitClient() error {
+	var rb RuntimeBinary
+
```

Comment:
We should look for a `HOF_CONTAINER_RUNTIME` variable here so the users can force a particular runtime, regardless of our fallback ordering.